orga/mirage/config.js

```diff
@@ -170,7 +170,7 @@ export default function() {
 
   this.get('/organizations/:id/students', findFilteredPaginatedStudents);
 
-  this.post('/organizations/:id/import-students', (schema, request) => {
+  this.post('/organizations/:id//schooling-registrations/import-siecle', (schema, request) => {
```

Comment:
Double slash dans l'URL `:id//school`

```suggestion
  this.post('/organizations/:id/schooling-registrations/import-siecle', (schema, request) => {
```

Reply:
done